test/unit/auth.js

```diff
@@ -7,6 +7,8 @@ import {oauthFlow, AccessToken} from '../../src/auth_v3'
 import mock from '../mock-api'
 import {decodeQuery} from '../../src/utils'
 
+global.navigator = undefined
```

Comment:
Why do you need to instantiate `gobal.navigator` to `undefined`.
If you don't declare `global.navigator`, then `global.navigator` is `undefined`...